lilypad/server/client.py

```diff
@@ -59,11 +59,11 @@ def __init__(
             )
         except FileNotFoundError:
             self.project_uuid = None
-
         if headers:
             self.session.headers.update(headers)
 
-        self._token = token if token else None
```

Comment:
what's going on here?

Reply:
when i set lilypad_client on initialization with token, it wasnt properly setting the authorization bearer token, so i had to update this

Reply:
i still don't quite understand why we need both `self._token` and `self.token` is `self._token` will always be `None`?